dandi/tests/test_dandiapi.py

```diff
@@ -59,6 +59,7 @@ def test_publish_and_manipulate(local_dandi_api, monkeypatch, tmp_path):
         },
     )
     dandiset_id = d.identifier
+    assert str(d) == f"DANDI {dandiset_id}/draft"
```

Comment:
hm, how do we get a space there?  I thought it would be just what we have in metadata
```
$> curl --silent https://api.dandiarchive.org/api/dandisets/000003/versions/0.210812.1448/ | jq .identifier               
"DANDI:000003"
$> curl --silent https://api.dandiarchive.org/api/dandisets/000003/versions/draft/ | jq .identifier     
"DANDI:000003"
```
where the divergence comes from?